src/LoginOrApp.js

```diff
@@ -18,16 +18,67 @@ class LoginOrAppComponent extends Component {
   }
 
   componentDidMount() {
-    this.props.fetchBootstrap(this.props.sessionState);
+    const dashboardName = this.getDashboardName();
+    this.props.fetchBootstrap(this.props.sessionState, dashboardName);
   }
 
+  getDashboardName = () => {
+    // gets dashboard name from url as defined in /admin/lizard_nxt/clientconfiguration/ "client slug" field
+    // the url should be in format of /dashboard/<dashboard_name>/
+
+    // if no dashboard name is given this function should return undefined
+    // this is required to create the basename of the JSX Router component (see in this file)
+
+    //  if a dashboard name of undefined is returned then the default name 'dashboard' will be used by function getBootstrap (in file actions.js).
+    // when no dashboard is in the url at all this function should return undefined. this should only be the case on dev
+
+    // if dashboardname = 'full' we assume it is not really the dashboardname but the url path to a full tile
+    // this rule should ensures that the url /dashboard/full/ returns  undefined
+
+    // examples:
+    // /dashboard/my_name/ -> my_name
+    // /dashboard/my_name -> my_name
+    // /dashboard/dashboard/ -> dashboard
+    // /dashboard/dashboard -> dashboard
+    // /dashboard/full/1 -> undefined
+    // /dashboard/ -> undefined
+    // /dashboard -> undefined
+    // / -> undefined
+
+    // split on first occurence of dashboard as in:
+    // https://stackoverflow.com/questions/4607745/split-string-only-on-first-instance-of-specified-character
+    const urlPostDashboard = window.location.href.split(/dashboard(.+)/)[1];
```

Comment:
Die `.+` is heel verwarrend, waarom is die nodig?

Reply:
Split werkt by default op alle occurences van het woord dashboard. Als de url dus bijvoorbeeld /dasboard/dashboard1/ is dan gaat de split ook de tweede dashboard spitten. Dit voorkom ik nu door een regex split. Zie ook het stack overflow antwoord erboven. 
Ben het wel met je eens dat het onduidelijk is. Sta ook open voor alternatieven. Of ik kan meer comments toevoegen

Reply:
Zie mijn wijziging hieronder.